crates/kubelet/src/lib.rs

```diff
@@ -47,25 +47,20 @@
 
 mod bootstrapping;
 mod kubelet;
-mod logs;
-mod node;
-mod pod;
-mod queue;
-mod server;
+
+pub(crate) mod kubeconfig;
+pub(crate) mod webserver;
 
 pub mod config;
+pub mod container;
 pub mod handle;
-pub mod image_client;
-pub mod module_store;
+pub mod image;
+pub mod log;
+pub mod node;
+pub mod pod;
 pub mod provider;
-pub mod status;
-pub mod volumes;
+pub mod store;
+pub mod volume;
 
 pub use self::kubelet::Kubelet;
 pub use bootstrapping::bootstrap;
-pub use handle::{LogHandleFactory, PodHandle, RuntimeHandle};
-pub use logs::{stream_logs, LogSendError, LogSender};
-pub use node::NodeBuilder;
-pub use pod::Pod;
-#[doc(inline)]
```

Comment:
Should we still inline the docs from `Provider` since it is such a critical component?

Reply:
The provider's now part of a separate module: `kubelet::provider::Provider`. I'll leave it up to you whether it makes sense to inline the provider documentation as part of module `kubelet`.